docs/docs/frigate/hardware.md

```diff
@@ -110,6 +110,14 @@ Frigate supports multiple different detectors that work on different types of ha
 | ssd mobilenet    | ~ 25 ms                         |
 | yolov5m          | ~ 118 ms                        |
 
+### AXERA
```

Comment:
I did not realize that synaptics was added here, but both of these should be moved down to the community supported detectors section below